src/posix/platform/platform-posix.h

```diff
@@ -83,7 +83,7 @@ enum
 };
 
 OT_TOOL_PACKED_BEGIN
-struct Event
+struct VirtualTimeEvent
```

Comment:
Can we turn this file in to a C++ header file? That would allow us to use namespaces.

Reply:
100% agree. I would say we should change the entire posix code to c++.
I would say we should tackle it in future PR. Here just renamed it to fix build errors (due to name conflicts).